packages/svelte/tests/helpers.js

```diff
@@ -117,8 +117,8 @@ export async function compile_directory(
 			}
 
 			const compiled = compile(text, {
-				outputFilename: `${output_dir}/${file}${file.endsWith('.js') ? '' : '.js'}`,
-				cssOutputFilename: `${output_dir}/${file}.css`,
+				outputFilename: `${file}${file.endsWith('.js') ? '' : '.js'}`,
+				cssOutputFilename: `${file}.css`,
```

Comment:
Don't know how I feel about this change. Arguably the outputFilename was more correct previously because it contained the full path to the file, which meant that the source maps where pointing to the correct relative location of their original - the Svelte 4 tests did this wrong, too, and it caused me a lot of headscratching trying to find out why some things were different which ultimately uncovered a bug in the Svelte 4 sourcemap filename handling.
AFAIK bundlers/tooling will set the full file path for the filename, too, so it might be better to keep this as-is and instead adjust the code generation to only use the last part of the path in some places (in others like the warning it's probably benefitial to have the full filename)

Reply:
Ideally I think `filename` would be cwd-relative, otherwise the message looks very chaotic (and in the worst case, if you somehow ship dev-mode code to production, will embed details of your filesystem in the deployment). We _could_ truncate the path at compile time but we don't know how much of it to truncate (unless we used `path.relative(cwd, filename)` which would add a dependency on Node that the compiler doesn't currently have)

Reply:
update to truncate absolute filenames